sphinx/source/docs/dev_guide/building.rst

```diff
@@ -211,9 +215,11 @@ not work correctly. The .exe will not be available within the Conda
 environment, which means you will use the version available in the base
 install, if it is available. Instead, you can make sure you use the version
 within the environment by explicitly running the bokeh-server python script
-in the root of the bokeh repository, similar to the following example::
+in the root of the bokeh repository, similar to the following example:
+
+.. code-block:: sh
 
-    python bokeh-server --script path\to\<yourapp>.py
+    $ python bokeh-server --script path\to\<yourapp>.py
```

Comment:
FWIW (personal opinion, not defined policy): I don't think there is anything in Bokeh that requires running sudo or root any longer so it's not too germane for this in particular. I also think that `$` vs `#` is a very subtle visual distinction and should not be relied upon anyway (better to state explicitly and emphasize: "run this as root") Also I think the benefit of not having it, is to make copy and pasting the commands much easier for users. That said, not putting it has been an adjustment for me too. :) I do think we have to mark `.. code-block: sh` explicitly now but that is fine with me, at least.